tests/Feature/Api/ExperienceSkillsControllerTest.php

```diff
@@ -124,4 +124,88 @@ public function testRestoreSoftDeletedExperienceSkill()
             ['id' => $experienceSkill->id, 'deleted_at' => null]
         );
     }
+
+    public function testBatchStoreExperienceSkill()
+    {
+        $work = factory(ExperienceWork::class)->create();
+        $workExperienceSkills = [
+            $this->makeExpSkillData($work->id, 'experience_work'),
+            $this->makeExpSkillData($work->id, 'experience_work'),
+            $this->makeExpSkillData($work->id, 'experience_work'),
+        ];
+        $response = $this->actingAs($work->experienceable->user)
+            ->json('post', route('api.v1.experience-skill.batchStore'), $workExperienceSkills);
+        $response->assertOk();
+        $response->assertJson($workExperienceSkills);
+        $ids = $response->decodeResponseJson('*.id');
+        $this->assertDatabaseHas(
+            'experience_skills',
+            [
+                'id' => $ids[0],
+                'id' => $ids[1],
+                'id' => $ids[2],
+            ]
+        );
+    }
+
+    public function testBatchUpdateExperienceSkill()
+    {
+        $faker = \Faker\Factory::create();
+        $work = factory(ExperienceWork::class)->create();
+        $experienceSkill1 = factory(ExperienceSkill::class)->create([
+            'experience_id' => $work->id,
+            'experience_type' => 'experience_work',
+        ]);
+        $experienceSkill2 = factory(ExperienceSkill::class)->create([
+            'experience_id' => $work->id,
+            'experience_type' => 'experience_work',
+        ]);
```

Comment:
Maybe explicitly set skill ids here? You could conceivably get the same skill twice (though unlikely).